Sources/KingfisherOptionsInfo.swift

```diff
@@ -84,13 +84,17 @@ public enum KingfisherOptionsInfoItem {
     /// It is the image scale, instead of your screen scale. You may need to specify the correct scale when you dealing 
     /// with 2x or 3x retina images.
     case scaleFactor(CGFloat)
-    
-    /// Whether all the GIF data should be preloaded. Default it false, which means following frames will be
-    /// loaded on need. If true, all the GIF data will be loaded and decoded into memory. This option is mainly
+
+    /// Whether all the animated image data should be preloaded. Default it false, which means following frames will be
+    /// loaded on need. If true, all the animated image data will be loaded and decoded into memory. This option is mainly
     /// used for back compatibility internally. You should not set it directly. `AnimatedImageView` will not preload
     /// all data, while a normal image view (`UIImageView` or `NSImageView`) will load all data. Choose to use
     /// corresponding image view type instead of setting this option.
-    case preloadAllGIFData
+    case preloadAllAnimationData
+
+    /// Back compatibility for deprecated preloadAllGIFData
+    @available(*, deprecated, renamed: "preloadAllAnimation")
```

Comment:
Could you move the `deprecated` statements to an extra extension at the end of this file (like what is done [here](https://github.com/onevcat/Kingfisher/blob/master/Sources/Image.swift#L867-L974))?